api/v1/verticadb_webhook.go

```diff
@@ -1618,6 +1609,43 @@ func (v *VerticaDB) checkImmutableSubclusterInSandbox(oldObj *VerticaDB, allErrs
 		}
 	}
 
+	newScInSandbox := v.GenSubclusterSandboxMap()
+	oldSbIndexMap := v.GenSandboxIndexMap()
+	oldSbMap := v.GenSandboxMap()
+
+	// This loop ensures a couple of things:
```

Comment:
Can demote/promote_subcluster endpoint eliminate this limitation?

Reply:
Yeah, it's possible this could be relaxed.